usort/util.py

```diff
@@ -77,6 +77,18 @@ def try_parse(path: Path, data: Optional[bytes] = None) -> cst.Module:
         raise parse_error or Exception("unknown parse failure")
 
 
+def parse_import(code: str) -> cst.SimpleStatementLine:
+    """
+    Parse a single import statement.
```

Comment:
I hope this is only for testing, can you clarify?  I'm afraid people might think splitting on newlines and calling this repeatedly is a good way to "speed up" parsing.